micropub.php

```diff
@@ -175,6 +174,10 @@ public static function parse_query( $wp ) {
 			if ( ! empty( $indieauth_scopes ) ) {
 				static::$scopes = $indieauth_scopes;
 			}
+			global $indieauth_token;
```

Comment:
huh. is this really an ok way for plugins to communicate?! seems like they should use hooks or filters instead?

Reply:
i do get that the code was already working this way, and this just extends that. still though.

Reply:
I am happy to see explore changibg it to a filter on both ends

Reply:
thanks! if you plan to switch to filters, feel free to merge this now and do that afterward.